### PR TITLE
Fix of the css loading bug on deployed page reload

### PR DESCRIPTION
**Pull request by mamadOuologuem**

### CODE REVIEW

File: src/components/Footer.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ import { Grid, Typography, Icon } from '@material-ui/core';
 import { makeStyles } from '@material-ui/styles';
 
 import { footerMenu } from '../services/menuItems';
-import { spacing, useStylesBase } from './styledComponents';
+import { spacing, stylesBase } from './styledComponents';
```
Comment (bencpeters) [inline src/components/Footer.js]: Let's destructure `stylesBase` here.

File: src/components/Footer.js [hunk 2]
```diff
@@ -25,8 +25,9 @@ const socialMedias = [
   },
 ];
 
+const { muiGridBlockContainerObj } = stylesBase;
 const useStyles = makeStyles({
-  muiGridBlockContainer: stylesBase.muiGridBlockContainer,
+  muiGridBlockContainer: muiGridBlockContainerObj,
```
Comment (bencpeters) [inline src/components/Footer.js]: In ES6, you can simplify this:

```javascript
const { muiGridBlockContainer } = stylesBase;
const useStyles = makeStyles({
  muiGridBlockContainer,
 ...
});
```

For object literals if you want the **key** to be the same name as the **variable** that your are assigning as the value, you can just write the name once, without the `key: value` syntax.

Comment (mamadOuologuem) [reply src/components/Footer.js]: Yea when I did that it was complaining about a double `const` declaration (second on the rendering function) so I had to renamed on of them.

Do you have a simpler way ?

Comment (bencpeters) [reply src/components/Footer.js]: Hrm, that's weird, the second `const` declaration should be scoped to the rendering function, they shouldn't conflict...

What was complaining, eslint? Or was it actually giving you a build error?

Comment (mamadOuologuem) [reply src/components/Footer.js]: eslint actually ... Can I ignore it ?

Comment (bencpeters) [reply src/components/Footer.js]: Did it say what rule it is referencing?

Comment (mamadOuologuem) [reply src/components/Footer.js]: yess
https://eslint.org/docs/rules/no-shadow

Comment (bencpeters) [reply src/components/Footer.js]: Maybe just add:

```javascript
//eslint-disable-next-line no-shadow
const {...}
```
That seems  a bit cleaner to me.

Comment (mamadOuologuem) [reply src/components/Footer.js]: Okey Dokey

**Comment by mamadOuologuem** [comment]
@ericboucher  PR mergable ?

**Pull request merged**